core/shared/src/test/scala/cats/parse/ErrorShowTest.scala

```diff
@@ -130,8 +139,9 @@ class ErrorShowTest extends munit.ScalaCheckSuite {
        |l4
        |ko
        |^
-       |* one of: {"ok"}
-       |* is: 'l'
+       |expectations:
+       |* must match one of the strings: {"ok"}
```

Comment:
this is kind of strange right? Maybe we should say `must match string "ok"` if there is only one item in that list?